bin/build

```diff
@@ -2,5 +2,5 @@
 
 set -e
 
-bin/scss-lint source/assets/stylesheets
+bin/scss-lint --config scss-lint.yml
```

Comment:
my bad, it works well with `scss_files` option in `scss-lint.yml`. That means that we don't need `source/assets/stylesheets` here: https://github.com/fs/static-base/pull/56/files#diff-a8da778d56fdb695e50daf67e50c0fbcR5